packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -42,7 +42,6 @@
   "force_start_apex_debug_logging": "SFDX: Turn On Apex Debug Log for Replay Debugger",
   "force_stop_apex_debug_logging": "SFDX: Turn Off Apex Debug Log for Replay Debugger",
   "isv_bootstrap_command_text": "SFDX: Create and Set Up Project for ISV Debugging",
-  "force_apex_log_get_text": "SFDX: Get Apex Debug Logs...",
```

Comment:
I removed the ellipsis from the text when i moved the message key over because as far as I can tell, most commands don't have this and it seemed inconsistent.